Sources/GRPCNIOTransportHTTP2Posix/Config+TLS.swift

```diff
@@ -167,6 +168,14 @@ extension HTTP2ServerTransport.Posix.TransportSecurity {
     /// use at that point in time.
     public var certificateReloader: (any CertificateReloader)?
 
+    /// Override the certificate verification with a custom callback that must return the verified certificate chain on success.
+    /// Note: The callback is only used when `clientCertificateVerification` is *not* set to `noVerification`!
+    public var customVerificationCallback:
```

Comment:
Can you annotate this new API with `@available(gRPCSwiftNIOTransport 2.2, *)`

You'll need to bump this number too: https://github.com/grpc/grpc-swift-nio-transport/blob/108495b705a68f6931281a96156532029ff8b9ed/Package.swift#L74